common/azure-common/src/main/java/com/azure/common/implementation/util/CloneUtils.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.azure.common.implementation.util;
+
+/**
+ *  The util class is a helper class for clone operation.
+ */
+
+public class CloneUtils {
```

Comment:
Don't have an empty line between the JavaDoc text and the code the comment is intended for. Also, I think this class name is too specific - perhaps 'ImplUtils' or similar is better. Finally, if you are making a util class, you should be sure to provide a private constructor so that people don't try to create an instance of the class.

Reply:
Good point with the naming and private constructor. Will do

Reply:
Remove the empty line. Renamed the file and implemented private constructor.